internal/core/component.rs

```diff
@@ -60,65 +60,56 @@ pub type ComponentRc = vtable::VRc<ComponentVTable, Dyn>;
 pub type ComponentWeak = vtable::VWeak<ComponentVTable, Dyn>;
 
 /// Call init() on the ItemVTable for each item of the component.
-pub fn init_component_items<Base>(
+pub fn init_component_items_array<Base>(
```

Comment:
I think the old name was fine. (This init the items, not the array)  not?

Reply:
I had to change the name since this one and the other one existed in parallel for a while.

But yes, renaming that back is a good idea.

Reply:
Renamed!